mono/mini/decompose.c

```diff
@@ -20,7 +20,6 @@
 #ifndef DISABLE_JIT
 
 /* FIXME: This conflicts with the definition in mini.c, so it cannot be moved to mini.h */
-MONO_API MonoInst* mono_emit_native_call (MonoCompile *cfg, gconstpointer func, MonoMethodSignature *sig, MonoInst **args);
```

Comment:
OK, will remove the fixme comment. Thought it was related to all three functions since they where grouped together.